### DISCUSSION TITLE
Wind Direction

### DISCUSSION THREAD

**Discussion by MET905** [niels747/2D-Weather-Sandbox#89] [Q&A]
How do we change the wind direction? I am trying to manipulate the direction of my thunderstorms.

**Comment by Meso22** [comment]
You can use the wind tool, or you could use the wind settings in the fluid settings.

**Reply by MET905** [reply]
Thank you for the reply, i know about the wind tool but i mean with the wind settings.

**Reply by Meso22** [reply]
If you set the wind in the negatives it will go left, and positive will go right.

**Reply by MET905** [reply]
thank you! One last question whats the best way to get extreme moisture levels? increase the water temps? I feel like if i let the sun bake the atmosphere for a few sim days and bring in a cold front i can make some really intense thunderstorms pop off.

**Reply by Meso22** [reply]
I make maps based off areas like Oklahoma and Texas, that means I don't have many large water sources, which is a problem, so I came up with a solution for that: 1. I make sure to have the map balanced out between vegetation and dry areas, the vegetation areas are for the moisture, and the dry areas are for convection and heat. 2. I set the Land Evap to completely full to allow strong dew points and high CAPE, I also set greenhouse gases to completely full so it can aid in the strength of the heat and moisture. 3. I make sure to have my sun intensity high enough so my map doesn't become over polluted with cold temps and moisture. I hope this helps!

**Reply by MET905** [reply]
Thank you! you have been a big help. just like i expected wicked thunderstorm development took place 
<img width="744" alt="STORM" src="https://github.com/niels747/2D-Weather-Sandbox/assets/113155608/a7e7eb47-e905-489c-8efd-5b9436353d18">

**Reply by Meso22** [reply]
Wow! that is a wild thunderstorm!
